spec/models/metric/ci_mixin/capture_spec.rb

```diff
@@ -36,6 +36,35 @@ def expected_stats_period_end
     parse_datetime('2013-08-28T12:41:40Z')
   end
 
+  context "#perf_capture_queue" do
+    it "splits up long perf_capture durations for old last_perf_capture_on" do
+      # there should be 10 days + a partial day, or 11 queue items
+      Timecop.freeze
+      original_queue_count = MiqQueue.count
```

Comment:
So was I.  It's because we queue up `stop_event_monitor` when we create OpenStack EMSes.  And, it just so happens that all this testing is relying on OpenStack mocks.

I thought about whacking it, but didn't wanna impact anything else.  I'm sure no one is relying on those existing queue messages, but ¯\\\_(ツ)\_/¯.